editions/tw5.com/tiddlers/widgets/EventCatcherWidget.tid

```diff
@@ -7,16 +7,18 @@ type: text/vnd.tiddlywiki
 ! Introduction
 
 <<.from-version "5.1.23">>
-The event catcher widget traps JavaScript events dispatched within its child content, and allows invoking a series of ActionWidgets in response to the events. 
 
-In order for the events to be trapped they must:
+//This is a technically advanced widget intended for use by those familiar with HTML, CSS and DOM event handling.//
```

Comment:
A few things here. "Technically advanced" is ambigious due to the word technically being used as an adverb. Suggest rewording it to: "This is an advanced technical widget" or dropping the word technical altogether.

As discussed at https://github.com/Jermolene/TiddlyWiki5/discussions/5321#discussioncomment-251219, the use of the term DOM without the context of JavaScript will be confusing to the layman user. We do not explain what DOM is and neither do I think we should. Therefore this needs to be expressed differently.

I suggest something along the lines of "by those familiar with HTML, CSS and JavaScript handling of DOM events".